dockers/docker-fpm-frr/frr/bgpd/gen_bgpd.conf.j2

```diff
@@ -0,0 +1,5 @@
+{% if DEVICE_METADATA.localhost.frr_mgmt_framework_config is defined and DEVICE_METADATA.localhost.frr_mgmt_framework_config == "true" %}
+    {% include "/usr/local/sonic/frr/bgpd.conf.j2" %}
```

Comment:
why not these in the /usr/share/sonic/templates/frrcfgd/ folder?

Reply:
This file is used by docker_init.sh of bgp docker to generate bgpd.conf. It's not specially related to frrcfgd.

Reply:
@lguohan "/usr/local" is the default target folder when installing python wheel package. Currently we put all template files used by frrcfgd only in sonic-frr-mgmt-framework package and install them as data files. So with this way it's not easy to copy them into "/usr/share" folder. Or we can move all template files from sonic-frr-mgmt-framework package to docker-fpm-frr and add command in Dockerfile to copy them to specified destination folder. Please suggest what is the preferred way.

Reply:
i see. them can you change the template folder to /usr/local/sonic/frrcfgd/ instead of /usr/local/sonic/frr?

Reply:
Yes, will make this change

Reply:
Addressed